app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php

```diff
@@ -55,24 +74,42 @@ public function __construct(
      */
     public function getStoreConfigData(StoreInterface $store): array
     {
-        $storeConfigData = array_merge(
-            $this->getBaseConfigData($store),
-            $this->getExtendedConfigData((int)$store->getId())
-        );
-        return $storeConfigData;
+        $defaultStoreConfig = $this->storeConfigManager->getStoreConfig($store);
+        return $this->prepareStoreConfigData($defaultStoreConfig);
     }
 
     /**
-     * Get base config data
+     * Get website available stores
      *
      * @param StoreInterface $store
      * @return array
      */
-    private function getBaseConfigData(StoreInterface $store) : array
+    public function getAvailableStoreConfig(StoreInterface $store): array
```

Comment:
Maybe this should take just the website id as an argument. We don't need anything else from the store, and it makes more sense to me since we're getting available stores for a particular website